jira/resources.py

```diff
@@ -1234,6 +1235,25 @@ def __init__(
         self.raw: dict[str, Any] = cast(Dict[str, Any], self.raw)
 
 
+class Team(Resource):
+    """A Jira team."""
+
+    def __init__(
+        self,
+        options: dict[str, str],
+        session: ResilientSession,
+        raw: dict[str, Any] = None,
+    ):
+        TEAM_API_BASE_URL = "{server}/gateway/api/public/teams/v1/"  # rest/{rest_path}/{rest_api_version}/{path}"
```

Comment:
The all caps naming convention does not need to be used for variables inside the init method, look at the `AgileResource` example which is more similar to what you intend to do here.